test/unit/test_couchdb_session_auth.py

```diff
@@ -168,6 +178,10 @@ def test_set_service_url(self):
 
 class TestCouchDbSessionAuthPatch(unittest.TestCase):
 
+    def setUp(self) -> None:
+        for key in os.environ:
+            del os.environ[key]
+
     def test_new_instance(self):
```

Comment:
Should we add a test for verifying `service` new instance with `disable_ssl_verification=True`?

Reply:
@emlaver something like https://github.com/IBM/cloudant-python-sdk/pull/342/commits/93cc7f1883d809867c3b94a4b9f1583ab42938bb ?

Reply:
Yea, https://github.com/IBM/cloudant-python-sdk/commit/93cc7f1883d809867c3b94a4b9f1583ab42938bb is what I was thinking.